src/modules/users/components/Inbox/InboxItem/InboxItem.css

```diff
@@ -115,4 +110,5 @@
 .avatarWrapper {
   display: inline-block;
   margin-right: 10px;
+  vertical-align: 8%;
```

Comment:
Why is this percentage based ? The spacing between top and left margins will always be the same no matter the height of the notification, or am I missing something

Reply:
No special reason, I just wanted to avoid writing a fractional `px`. Hope that's okay.